internal/agent/testdata/device.yaml

```diff
@@ -18,7 +18,7 @@ spec:
             - contents:
                 source: >-
                   data:,This%20system%20is%20managed%20by%20flightctl.%0A
-              mode: 600
+              mode: 0600
```

Comment:
pretty sure I copied this from examples so we should update that can be a followup

Reply:
Yes, we have the error all around. I was crawling inside the ignition config specs and docs, it seems like in the json formating octal is not even supported and you need to write the decimal version of that octal (384) for it to work .

At least in yaml we can use 0xxx